docs/getting-started-guides/kubeadm.md

```diff
@@ -177,12 +177,43 @@ You must install a pod network add-on so that your pods can communicate with eac
 
 **It is necessary to do this before you try to deploy any applications to your cluster, and before `kube-dns` will start up. Note also that `kubeadm` only supports CNI based networks and therefore kubenet based networks will not work.**
 
-Several projects provide Kubernetes pod networks using CNI, some of which 
-also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-ons page](/docs/admin/addons/) for a complete list of available network add-ons.
-
-You can install a pod network add-on with the following command: 
-
-    # kubectl apply -f <add-on.yaml>
+Several projects provide Kubernetes pod networks using CNI, some of which also support [Network Policy](/docs/user-guide/networkpolicies/).
+
+You must select a network addon from the drop-down menu below, and correct instructions will appear below.
```

Comment:
Sorry to be late to this thread.  I'm not a fan of this drop down idea. Mainly because I can't' recall ever seeing it used in docs to address this kind of problem? It seems like a strange thing to see on a doc page. Are there examples of this that someone could point me to? 

Also, it doesn't address the problem of enumerating all the options because every add-on provider will want to be include in the drop down. Finally, it introduces additional doc edit/management complexity.  So, in addition to markdown, I have to know javascript to make a change?  Some of my trivially simple markdown edits have broken the rendering of some pages, and IMO adding scripts is opening a real bag of worms.

That said, happy to defer to doc experts on this.